app/uk/gov/hmrc/errorhandling/ProcessingError.scala

```diff
@@ -16,14 +16,18 @@
 
 package uk.gov.hmrc.errorhandling
 
-import uk.gov.hmrc.personaldetailsvalidation.model.QueryParamConverter
+sealed trait ProcessingError {
+  def message: String
 
-case class ProcessingError(message: String)
+  override def toString: String = message
 
-object ProcessingError {
-  implicit val queryParamCOnverter: QueryParamConverter[ProcessingError] = new QueryParamConverter[ProcessingError] {
-    override def toQueryParam(queryParam: ProcessingError) = {
-      Map("technicalError" -> Seq(""))
-    }
-  }
+  def toQueryParam: Map[String, Seq[String]] = ???
```

Comment:
??? ?

Reply:
Undefined defs are abstract by default, ??? just means it won't demand an implementation and would then go bang at runtime

https://github.com/scala/scala/blob/v2.12.12/src/library/scala/Predef.scala#L288